### PR TITLE
iss1: style variables

### PR DESCRIPTION
**Pull request by krelo1320**

### CODE REVIEW

File: themes/marvelous-snuffle/static/css/style.css [hunk 1]
```diff
@@ -1,39 +1,51 @@
 /* ============================================================= COLORS ** ============================================================= */
+:root{
+    --global-font-color:  #e0e0e0;
```
Comment (not7cd) [inline themes/marvelous-snuffle/static/css/style.css:3]: this should be `body-color`

File: themes/marvelous-snuffle/static/css/style.css [hunk 2]
```diff
@@ -1,39 +1,51 @@
...
+    --global-button-background-color: #283747;
+    --background-color-2: #212121;
+    --global-border-color: var(--global-font-color);
+    --global-border: 1px solid var(--global-border-color);
+    --global-font-size: 13px;
+    --global-padding: 15px;
+    --fill-color: var(--global-button-background-color);
+    --active-color: #0c99
```
Comment (not7cd) [inline themes/marvelous-snuffle/static/css/style.css:11]: this should be `primary` color, we can also add `secondary` and other colors this way.
please look at https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss#L69

File: themes/marvelous-snuffle/static/css/style.css [hunk 3]
```diff
@@ -1,39 +1,51 @@
 /* ============================================================= COLORS ** ============================================================= */
+:root{
+    --global-font-color:  #e0e0e0;
+    --global-button-background-color: #283747;
+    --background-color-2: #212121;
+    --global-border-color: var(--global-font-color);
+    --global-border: 1px solid var(--global-border-color);
+    --global-font-size: 13px;
+    --global-padding: 15px;
```
Comment (not7cd) [inline themes/marvelous-snuffle/static/css/style.css:9]: Please don't use px as a unit, I would recommend em, or rem. Which will be more responsive.

File: themes/marvelous-snuffle/static/css/style.css [hunk 4]
```diff
@@ -1,39 +1,51 @@
 /* ============================================================= COLORS ** ============================================================= */
+:root{
+    --global-font-color:  #e0e0e0;
+    --global-button-background-color: #283747;
+    --background-color-2: #212121;
```
Comment (not7cd) [inline themes/marvelous-snuffle/static/css/style.css:5]: What is that

**Review by not7cd — CHANGES_REQUESTED** [review]
I belive this is a good inspiration on how variables should be named.
https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss

File: themes/marvelous-snuffle/static/css/style.css (hunk 1 above)
Comment (krelo1320) [reply themes/marvelous-snuffle/static/css/style.css]: Ok

File: themes/marvelous-snuffle/static/css/style.css (hunk 2 above)
Comment (krelo1320) [reply themes/marvelous-snuffle/static/css/style.css]: ok, thx

File: themes/marvelous-snuffle/static/css/style.css (hunk 3 above)
Comment (krelo1320) [reply themes/marvelous-snuffle/static/css/style.css]: Ok

File: themes/marvelous-snuffle/static/css/style.css (hunk 4 above)
Comment (krelo1320) [reply themes/marvelous-snuffle/static/css/style.css]: Removed

**Comment by not7cd** [comment]
przykryte przez #6

**Pull request closed without merging**